assets/app/styles/_core.less

```diff
@@ -1081,3 +1081,8 @@ h1 small.meta, .build-config-summary .meta {
   vertical-align: middle;
   margin-left: 2px;
 }
+
+// don't allow copy-to-clipboard .input-groups get bigger than 600px so Github webhook secrets aren't revealed within the input-group
+copy-to-clipboard .input-group {
+  max-width: 600px;
```

Comment:
I wonder if we shouldn't target just the webhooks (and other copy-to-clipboard uses on the same page) instead of applying max-width to all uses.